docs/install-commcare-sync.md

```diff
@@ -1,65 +1,99 @@
-Production Environments
-=======================
+Set up CommCare Sync
+=====================
+This page outlines the process you need to follow in order to set up a CommCare Sync instance on a machine.
 
-This page describes creating and managing production inventory files.
-For details on deploying a production environment, see the [deployment page](/deployment).
+## Install Ansible
 
-### Directory overview
+From the [Ansible Installation Guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-on-ubuntu),
+run the following on your local/control machine
 
-Production environments live in the `inventories/` folder.
+```bash
+sudo apt-add-repository ppa:ansible/ansible
+sudo apt-get update
+sudo apt-get install ansible
+```
+
+## Install CommCare Sync
+
+### Choose a location for the control
```

Comment:
Aren't these docs sort of specific to production?

In development I'd think the control is always the dev machine and the "host" is always the vagrant VM (which is different from the guidance below)